testdata/extension.config.yml

```diff
@@ -23,6 +23,23 @@ extensions:
       resource:
         url: 'cart/1234'
   - uuid: 00000000-0000-0000-0000-000000000001
+    type: checkout_ui_extension_next
```

Comment:
I'm working on `dev` and just started running into these errors as well. Since that's next on my list to solve, I think I'll take it on and fix. And once it's done, I'll add the fix to this PR.

Reply:
Took me all day but I fixed this issue. We'd forgotten to use `NodeExecutable` in `develop` mode, and the config wasn't quite right. It seems to work fine for me locally now.

I believe the other issue raised will be handled by a PR to `shopify-cli`, so @vividviolet are we good to merge and continue?